tests/main/progress/task.yaml

```diff
@@ -0,0 +1,41 @@
+summary: Verify progress reporting from snap command
+
+details: |
+  Verify that progress reporting from snap command works as expected depending
+  on whether the command is ran with a tty attached or not.
+
+systems: [-ubuntu-core-*]
+
+prepare: |
+  tests.pkgs install socat
```

Comment:
Should this have a `tests.pkg remove ...` when restoring?

Reply:
no, the cleanup of new packages installed by tests is done automatically in restore.